CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift

```diff
@@ -145,6 +145,28 @@ import GiniBankAPILibrary
                                 borderWidth: 0,
                                 shadowRadius: 0,
                                 withBlurEffect: false)
+
+    // MARK: - Navigation Button Configuration
+
+    /**
+     Sets custom icon for the every Cancel button on the top navigation bar
+     */
+    public var giniNavigationBarButtonCancelButtonIcon: UIImage?
+
+    /**
+     Sets custom icon for the every Help button on the top navigation bar
+     */
+    public var giniNavigationBarButtonHelpButtonIcon: UIImage?
+
+    /**
+     Sets custom icon for the every Back button on the top navigation bar
+     */
+    public var giniNavigationBarButtonBackButtonIcon: UIImage?
```

Comment:
[Clarification] will it work if we set in the SDK default image.asset with an empty image and if the user decides to override it then the image will be added to the nav button?🤔

Reply:
It worked with this approach. :D Thx for the idea 🙏